detox/android/detox/src/main/java/com/wix/detox/LaunchIntentsFactory.kt

```diff
@@ -0,0 +1,102 @@
+package com.wix.detox
+
+import android.app.Activity
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+
+internal class LaunchIntentsFactory {
```

Comment:
Extracted from `Detox`, alongside the additional `intentWithNotificationData` method that is used in the notification-data handling flow